### PR TITLE
Issue 295: Identity, Tags, CloudWatch Log Group v0.11

### PR DESCRIPTION
**Pull request by dannyleesmith**
##### Corresponding Issue(s):

https://github.com/rackspace-infrastructure-automation/aws-terraform-internal/issues/295

##### Summary of change(s):

- Add identity output
- Use tags
- Managed CloudWatch Log Group

##### Reason for Change(s):

- QOL

##### Will the change trigger resource destruction or replacement? If yes, please provide justification:

No

##### Does this update/change involve issues with other external modules? If so, please describe the scenario.

No

##### If input variables or output variables have changed or has been added, have you updated the README?

Updated

##### Do examples need to be updated based on changes?

Done

##### Note to the PR requester about Closing PR's

Please message the person that opened the issue when auto closing it on slack, as well as any other stake holders of deep interest. Only close the issue if you believe that the issue is fully resolved with this PR.

#### This PR may auto close the issue associated with it. If you feel the issue is not resolved please reopen the issue.

### CODE REVIEW

File: modules/cluster/examples/example.tf [hunk 1]
```diff
@@ -1,10 +1,10 @@
 provider "aws" {
-  version = "~> 2.7"
+  version = "~> 2.52"
```
Comment (twistedgrim) [inline modules/cluster/examples/example.tf]: Is there a reason to bump this down to 2.52?  Having it at 2.7 is the min required for TF.12 so its prob better to keep it at that unless theres a good reason.

Comment (dannyleesmith) [reply modules/cluster/examples/example.tf]: This is bumping it up to the latest provider release, not bumping it down. Just felt like a cleanliness thing.

File: modules/cluster/examples/example.tf [hunk 2]
```diff
@@ -14,11 +14,11 @@ locals {
 module "vpc" {
   source = "git@github.com:rackspace-infrastructure-automation/aws-terraform-vpc_basenetwork?ref=v0.0.10"
 
-  vpc_name = "Test1VPC"
-
   custom_tags = {
     "kubernetes.io/cluster/${local.eks_cluster_name}" = "shared"
   }
+
+  vpc_name = "Test1VPC"
```
Comment (johnctitus) [inline modules/cluster/examples/example.tf]: Nitpicky, but we should revert this change, as multiline properties should come after all single line properties.

File: modules/cluster/examples/example.tf [hunk 3]
```diff
@@ -69,4 +61,12 @@ module "ec2_asg" {
       propagate_at_launch = true
     },
   ]
+
+  ec2_os                    = "amazoneks"
```
Comment (johnctitus) [inline modules/cluster/examples/example.tf]: We should also revert this change based on single line vs multi-line properties.

File: modules/kubernetes_components/README.md [hunk 4]
```diff
@@ -1,37 +1,48 @@
-# aws-terraform-eks/modules/kubernetes_components
+# aws-terraform-eks/modules/kubernetes\_components
 
 This module creates the other required components for EKS to allow additional features like ALB Ingress and Cluster Autoscaler.
 
-## Basic Usage
+\*## Basic Usage
```
Comment (johnctitus) [inline modules/kubernetes_components/README.md]: Nothing to do with your change, but looks like the latest version of terraform-docs has mangled this rendered readme.  Looks like we need to add spaces between the `*` and the start of the header line in `modules/kubernetes_components/main.tf`

File: modules/cluster/examples/example.tf (hunk 2 above)
Comment (dannyleesmith) [reply modules/cluster/examples/example.tf]: In Terraform resources that's true but in Rackspace process standards when we use Rackspace modules we try to keep the vars in alphabetical order so it matches with the README.md

File: modules/cluster/examples/example.tf (hunk 3 above)
Comment (dannyleesmith) [reply modules/cluster/examples/example.tf]: In Terraform resources that's true but in Rackspace process standards when we use Rackspace modules we try to keep the vars in alphabetical order so it matches with the README.md

File: modules/kubernetes_components/README.md (hunk 4 above)
Comment (dannyleesmith) [reply modules/kubernetes_components/README.md]: Ah yes, I had only worked on the one module, forgot to check the header comment was formatted properly for rendering

File: modules/cluster/examples/example.tf (hunk 2 above)
Comment (johnctitus) [reply modules/cluster/examples/example.tf]: Might be worth further discussion offline, but until that can occur and the defined style guidelines are updated, I would prefer we continue to treat all resources (module, explicit resource, and data resource) the same way within the modules.

Comment (dannyleesmith) [reply modules/cluster/examples/example.tf]: As discussed, reverted

File: modules/cluster/examples/example.tf (hunk 3 above)
Comment (dannyleesmith) [reply modules/cluster/examples/example.tf]: As discussed, resolved
